### PR TITLE
Fix #1872 - Closed Center, Group& Clients are shown on User Interface

### LINKED ISSUE #1872: Closed Center, Group& Clients are shown on User Interface
Reported by Amit Jain at https://mifosforge.jira.com/browse/MIFOSX-2109

Original Description:
1.If we closed any Center, Group & Client even any loan it Should not comes on user Interface 2.Presently it comes on User Interface.
3.I proposed a "Show All" button on UI it works when a user need to view details including closed items.
4. In reports the balance shown is 0 always when the account has been closed I think it should not comes in reports.After a certain level of organisation it creates panic.

closed client

### PR DESCRIPTION
**Pull request by MYKatz**
## Description
This fix hides closed centers, groups and clients from their respective pages.  Pressing the "show all" button shows the closed items.

## Related issues and discussion
#1872 

## Screenshots, if any
untitled

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [x] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

(Powered by [GitMate.io](https://gitmate.io))

**Comment by santoshmath** [comment]
@mbj36, @gkrishnan724 ,FYI

 https://codein.withgoogle.com/dashboard/task-instances/5135323111620608

**Comment by gkrishnan724** [comment]
@MYKatz Change commit message to `Fix #1872 - Hide closed groups,centers,clients`

File: app/views/centers/centers.html [hunk 1]
```diff
@@ -18,6 +18,9 @@
                                     <span class="fa fa-search"></span>
                                 </button>
                             </span>
+                            <span class="btn btn-primary form-control" ng-click="showAll()">
+                                {{'label.button.showall' | translate}}
```
Comment (gkrishnan724) [inline app/views/centers/centers.html]: I suggest this to be a checkbox named `show Closed`

File: app/views/clients/clients.html [hunk 2]
```diff
@@ -30,13 +30,15 @@ <h4>List of Clients</h4>
                                             <span class="fa fa-search"></span>
                                         </button>
                                     </span>
+                                    <span class="btn btn-primary form-control" ng-click="showAll()">
+                                            {{'label.button.showall' | translate}}
```
Comment (gkrishnan724) [inline app/views/clients/clients.html]: same here

File: app/views/groups/groups.html [hunk 3]
```diff
@@ -23,6 +23,9 @@
                                     <span class="fa fa-search"></span>
                                 </button>
                             </span>
+                            <span class="btn btn-primary form-control" ng-click="showAll()">
+                                            {{'label.button.showall' | translate}}
```
Comment (gkrishnan724) [inline app/views/groups/groups.html]: here too

**Comment by MYKatz** [comment]
@gkrishnan724 Made the changes, thanks!

**Comment by gkrishnan724** [comment]
Please make the label `show Closed` since enabling that should show the closed accounts too

**Pull request merged**